docs/devices/7099860PH.md

```diff
@@ -19,25 +19,38 @@ description: "Integrate your Philips 7099860PH via Zigbee2MQTT with whatever sma
 
 
 ### Pairing
+
+Philips LivingColors Devices that come with
+Philips (HUE) Remote Gen 2 (Touch Wheel) or Gen 3 (Round Click Wheel) can be paired via Zigbee.
+Devices with Gen 1 Remote don't use Zigbee and can not be paired.
+The Philips LivingColors Remote can not be paired via Zigbee because it only support ZigBee Light Link (ZLL).
+
+To Pair hold Button ON and Bottom Left Key (Favorite 1) on the Remote in Front of the Device until
+the Device Light blinks and lights Orange. If connection was successful the Device Light will light Green.
+
+**WARNING**: If you pair your Device to a Zigbee Network which is not using a ZLL Channel
+you can't reset the Device with the Philips LivingColors Remote Gen 3.
+The Gen 3 Remote will only try ZLL Channels to find the Device! Maybe it's possible to reset
+the Device with a Philips LivingColors Remote Gen 2 as it should try all Zigbee Channels to find the Device.
+
+[Philips LivingColors Bloom Manual](https://www.download.p4c.philips.com/files/7/7099760pu/7099760pu_dfu_eng.pdf)
+
+[Philips LivingColors Iris Manual](https://www.download.p4c.philips.com/files/7/7099930ph/7099930ph_dfu_eng.pdf)
+
 Factory resetting a Hue bulb can be accomplished in 5 ways.
 After resetting the bulb will automatically connect.
 
-#### Touchlink factory reset
+#### Touchlink factory reset (not tested)
```

Comment:
I didn't want to radically change the whole page, but maybe this isn't such a bad idea after all. Most of the text applies to "regular" Hue bulbs/lamps, not the LivingColors Trademark.

I'm no expert on this topic, but what I figured so far:  
The "Philips LivingColors" trademark came before the Hue in time. They used solely these kind of remotes for control (based on this "ZigBee Light Link" - whatever the difference is):
![](https://hueblog.de/wp-content/uploads/2018/01/fernbedienung-genii.jpg)

Starting with lamps that were sold with Gen2 or Gen3 remotes, the lamps supported the "regular" Hue-ZigBee as well, and in fact they could be paired with an official Hue Bridge with [some (undocumented) key combination](https://www.youtube.com/watch?v=x_CtvhXAb5E) on the LivingColors Remote. So even though the remote itself could not be paired with the Hue Bridge (because it uses ZLL or something), it did send some kind of "hey lamp, please search for ZigBee networks now"-signal.  
From my experience there is no other way to pair a LivingColors lamp with the Hue Bridge. Which is kind of unfortunate because the remotes break annually (dunno, they're just bad quality). So you need the LivingColors Remote once and then you can control the lamps via the Hue Bridge.

However I also found out, that it's possible to reset a LivingColors with the new Hue dimmer button ([this video](https://www.youtube.com/watch?v=qvlEAELiJKs)) - which is bad if you didn't test your old LivingColors remote before resetting the lamp. Well.. yeah.. I always wanted to buy a new one anyway.

Reply:
And yeah, I can't verify if it's possible to pair a LivingColors with zigbee2mqtt, because my old remote doesn't work anymore, but there are people who say it's possible to pair them with z2m, however one of them makes references to pages that are not available anymore:

- https://github.com/Koenkk/zigbee2mqtt/issues/375
- https://github.com/Koenkk/zigbee2mqtt/issues/1198#issuecomment-471239619

Reply:
In that case I propose to remove it, can you update the page?

Reply:
Sure.
How do you feel about splitting the 'basic' instructions of Philips Hue and LivingColors to a seperate document? We could make a link in every compatible product like "Generic instructions >here<" or something.  
This way it's much clearer which are the individual (and thus confirmed) device-specific options.

Reply:
@dersimn I don't think we should go that way. I prefer to remove anything under **Pairing** that was not confirmed to be working. Thus leaving only LivingColors specific instructions here.